samples/beam/hello_world_write_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import uuid
+
+from google.cloud import bigtable
+import pytest
+
+import hello_world_write
+
+PROJECT = os.environ['GOOGLE_CLOUD_PROJECT']
+BIGTABLE_INSTANCE = os.environ['BIGTABLE_INSTANCE']
```

Comment:
Where/how is this environment variable being set in the test environment?

Reply:
These are the same variables I've used for other tests, so I assumed they were still available since all the other snippets that were moved are still working. 

They should be defined during [test-samples.sh](https://github.com/googleapis/python-bigtable/blob/master/.kokoro/test-samples.sh#L53) 
They are stored in a secret in the cloud and then [decrypted](https://github.com/googleapis/python-bigtable/blob/master/scripts/decrypt-secrets.sh)